action.yaml

```diff
@@ -68,14 +68,15 @@ runs:
       shell: bash
       if: inputs.github-token != 'false'
       run: |
-        git remote set-url origin https://$DEPLOY_ACTOR:$DEPLOY_TOKEN@github.com/rentpath/${REPOSITORY}
+        git remote set-url origin https://$DEPLOY_ACTOR:$DEPLOY_TOKEN@github.com/${OWNER_AND_REPOSITORY}
```

Comment:
Note that if `skip_checkout` is `false`, then the origin url is already set and the checkout action has already authenticated. However, the next line is key to rewrite _all_ git urls (including, say, the `ex_ops` one).